rasa/core/domain.py

```diff
@@ -31,6 +31,11 @@
 
 PREV_PREFIX = "prev_"
 ACTIVE_FORM_PREFIX = "active_form_"
+DEFAULT_SESSION_LENGTH = 60
```

Comment:
```suggestion
DEFAULT_SESSION_LENGTH = 60  # in minutes
```
or even call it `DEFAULT_SESSION_LENGTH_IN_MINUTES`?

Reply:
and this should probably live in `constants.py`